whois-commons/src/main/resources/patch/whois-1.197.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS `transfer_update_lock`;
```

Comment:
why this?

Reply:
I see we have that table in WHOIS_UPDATE_RIPE.
OK

Reply:
because we are moving this table to internals so delete this table from whois schema